### ISSUE TITLE
Are Maven artifacts available?

### ISSUE THREAD

**Issue by ejschoen** [carlspring/s3fs-nio#577]
# Bug Description

Deployment missing on Maven

# Steps To Reproduce

This coordinate, from the getting started, doesn't exist:

<dependency>
    <groupId>org.carlspring.cloud.aws</groupId>
    <artifactId>s3fs-nio</artifactId>
    <version>1.0-SNAPSHOT</version>
</dependency>

# Expected Behavior

Able to depend upon s3fs-nio from org.carlspring.cloud.aws.

Hi!  Thanks for working on s3fs-nio.  A recent high-severity CVE on the version of the AWS Java SDK used in Upplication's version of s3fs is driving us to look for a newer version, and your version looks new enough to not be exposed to the CVE.  But there are no Maven artifacts for the project, and I'd rather not have to build and manage it itself.

**Comment by carlspring** [comment]
Hi Eric,

You can build the code from source quite easily, as explained in the build instructions.

In addition, I would like to clarify that the code in this repository is significantly more up-to-date, than the one in Upplication.

**Comment by stefanofornari** [comment]
Still the question is relevant: any plan to publish to maven central the artifact?

**Comment by carlspring** [comment]
Hi @stefanofornari ,

Yes, we are actually going to cut a release within the next few days. We've just set up a release pipeline and migrated from Maven to Gradle.

**Comment by stefanofornari** [comment]
ok great, thanks! would it be possible to keep also maven build ?
Ste

**Comment by steve-todorov** [comment]
> @stefanofornari  Still the question is relevant: any plan to publish to maven central the artifact?

Thanks for the interest in the project. You are correct this is still relevant. Our plans are to do a release `1.0.X` within next week. 
We are currently in the process of finalizing the pipeline to automate this.

You can immediately checkout [`1.0.0-SNAPSHOT`](https://oss.sonatype.org/content/repositories/snapshots/org/carlspring/cloud/aws/s3fs-nio/1.0.0-SNAPSHOT/maven-metadata.xml) which is the `master` branch version. 

<details>
  <summary>pom.xml</summary>
  ```
  <dependency>
      <groupId>org.carlspring.cloud.aws</groupId>
      <artifactId>s3fs-nio</artifactId>
      <version>1.0.0-SNAPSHOT</version>
  </dependency>
  ```
</details>

<details>
  <summary>gradle.build.kts</summary>
  ```
  implementation("org.carlspring.cloud.aws:s3fs-nio:1.0.0-SNAPSHOT")
  ```
</details>

**Comment by carlspring** [comment]
@ejschoen , @stefanofornari ,

Thanks for your interest! We have now released version [1.0.0](https://github.com/carlspring/s3fs-nio/releases/tag/v1.0.0) to Maven Central (https://repo.maven.apache.org/maven2/).
